.drone.yml

```diff
@@ -114,15 +114,17 @@ build:
       - export WORKER1=$VM2
       - export WORKER2=`$GOVC_GET_IP Ubuntu.14.04`
       - #
-      - ./misc/drone-scripts/deploy-and-test-wrapper.sh deployplugin $GOVC_URL_6_5 $VM1 $VM2 $$BUILD_NUMBER < /dev/null
+      - export CI_NODE4=`$GOVC_GET_IP photon.vsan`
```

Comment:
How are we going to use this new VM? Just want to know if we can have a better name for this VM based on the usage info (what tests need this new VM?).